config/environments/development.rb

```diff
@@ -7,7 +7,7 @@
   config.cache_classes = false
 
   # Do not eager load code on boot.
-  config.eager_load = false
+  config.eager_load = true
```

Comment:
@RyanNovas do we know why anyone would want this to be false to begin with?
Seeing here - https://blog.pivotal.io/labs/labs/rails-4-upgrade
that they used .eager_load=true in production.rb